src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/VilkårperiodeController.kt

```diff
@@ -33,6 +34,13 @@ class VilkårperiodeController(
         return vilkårperiodeService.hentVilkårperioderDto(behandlingId)
     }
 
+    @GetMapping("behandling/{behandlingId}/v2")
+    fun hentVilkårperioder(@PathVariable behandlingId: UUID): VilkårperioderResponse {
+        tilgangService.validerTilgangTilBehandling(behandlingId, AuditLoggerEvent.ACCESS)
+
+        return vilkårperiodeService.hentVilkårperioderResponse(behandlingId)
+    }
```

Comment:
Her hadde man kunnet lagt inn 
`val grunnlag: VilkårperioderGrunnlag,` i `VilkårperioderDto` og, uten å endre endepunkt.
Men har ikke veldig mye å si.

```kotlin
data class VilkårperioderDto(
    val målgrupper: List<VilkårperiodeDto>,
    val aktiviteter: List<VilkårperiodeDto>,
    val grunnlag: VilkårperioderGrunnlag,
)
// vs
data class VilkårperioderResponse(
    val vilkårperioder: VilkårperioderDto,
    val grunnlag: VilkårperioderGrunnlag,
)
```

Reply:
Forsøkte det i starten, men det ble så mange omskrivinger. Da måtte `Vilkårperioder.tilDto()` tatt inn grunnlag, og da må vi hente opp grunnlaget fra databasen mange steder vi egentlig ikke er interessert i det. Vi bruker på mange måter dto'en som et domeneobjekt internt i ts-sak, så vi må mappe til dto en del forskjellige steder 😕

Reply:
Ah ja, ulempen med at vi sender runt VilkårperioderDto en del steder 😁

Reply:
Syns egt det her var en bedre modell, så vi kan jo ha den i hodet for en evt. omskriving av vilkårperiode-konseptet senere. Er jo en del krøll der 😢